components/form/SecretSelector.vue

```diff
@@ -144,6 +144,18 @@ export default {
     display: block;
   }
 
+  & .labeled-select {
+    min-height: 61px;
```

Comment:
There's a scss variable for form field height (`$input-height`) that should be used here